components/OperationFunction.js

```diff
@@ -25,15 +33,24 @@ export function GenerateReceiveFunctions({ operations }) {
   receiveDetails.forEach(t => {
     // Generate receive function
     functions += `def ${t.functionName}(self, callback):
-        topic = "${t.topic}"
-        self.client.subscribe(topic)
-        self.client.message_callback_add(topic, callback)\n`;
+    """
+    Subcribes to a topic and adds a message callback to handle received messages.
```

Comment:
modify `getFunctionDetails` so it returns object not only with `functionName` and `topic` but also with `summary` (that you first need to add to asyncapi document, the test one we have, to each operation)

this way in this code, instead of hardcoding `Subcribes to a topic and adds a message callback to handle received messages.` you can just do `${t.summary}`

Reply:
Added the modifications. The current ` Client.py` file looks like this:

```python

import paho.mqtt.client as mqtt

mqttBroker = "test.mosquitto.org"

class CommentsServiceClient:

  def __init__(self):
              self.client = mqtt.Client()
              self.client.connect(mqttBroker)

  def sendCommentLiked(self, id):
      """
      Args: 
      self: CommentsServiceClient
      id: Comment ID
      """
      topic = "comment/liked"   
      summary = "Message sent to the broker when a comment is liked"
      self.client.publish(topic, id)
  def sendCommentUnliked(self, id):
      """
      Args: 
      self: CommentsServiceClient
      id: Comment ID
      """
      topic = "comment/unliked"   
      summary = "Message sent to the broker when a comment is unliked"
      self.client.publish(topic, id)
  def receiveCommentViews(self, callback):
      """
      Args:

      self:  CommentsServiceClient
      callback: Callback function to handle received messages
      """
      topic = "comment/views"
      summary = "Message received when a comment is viewed"
      self.client.subscribe(topic)
      self.client.message_callback_add(topic, callback)

  def loop(self): 
          '''This is a blocking form of the network loop and will not return until the client calls disconnect(). It automatically handles reconnecting.'''
          self.client.loop_forever()
```